lang/it.json

```
@@ -66,19 +66,19 @@
   "Uniform": "Uniforme",
   "Dropshadow": "Ombreggiatura",
   "Font Family": "Famiglia di caratteri",
-  "Proportional Sans-Serif": "Carattere a spaziatura variabile senza grazie (Proportional Sans-Serif)",
-  "Monospace Sans-Serif": "Carattere a spaziatura fissa senza grazie (Monospace Sans-Serif)",
-  "Proportional Serif": "Carattere a spaziatura variabile con grazie (Proportional Serif)",
-  "Monospace Serif": "Carattere a spaziatura fissa con grazie (Monospace Serif)",
+  "Proportional Sans-Serif": Sans-Serif proporzionale",
```

```suggestion
  "Proportional Sans-Serif": "Sans-Serif proporzionale",
```
missing a quote here

---

Whoops! sorry, I'll amend my last commit

---

Done @mister-ben